test/lit/metadce/sourcemap.wat

```diff
@@ -14,13 +14,3 @@
      ;;@ a:3:1
   )
 )
-;; CHECK:      (type $0 (func))
-
-;; CHECK:      (export "f" (func $0))
-
-;; CHECK:      ;;@ a:1:1
-;; CHECK-NEXT: (func $0
-;; CHECK-NEXT:  ;;@ a:2:1
-;; CHECK-NEXT:  (nop)
-;; CHECK-NEXT:  ;;@ a:3:1
-;; CHECK-NEXT: )
```

Comment:
It looks like this test lost all its testing - should it be a manual test perhaps? But why is this CHECK output vanishing - is it hitting an auto updater bug perhaps?

Reply:
It looks like the function just needs to be named so that the update script can recognize it and match it up with the output. That's enough to test everything except the prologue source comment, which the update script does not know to emit.